packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -472,6 +474,7 @@ export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldSt
         onChange={this._onInputChange}
         className={this._classNames.field}
         aria-label={this.props.ariaLabel}
+        aria-labelledby={this._labelId}
```

Comment:
Do we want to provide this even if `label` prop is not provided? I think that could generate an ARIA accessibility error.

Reply:
This would be good to make sure there's a snapshot test for.. a `TextField` with no `label` and verify the snapshot output does not show `aria-labelledby` in the output. 😄

Reply:
Also one other thing, if the `aria-labelledby` prop is provided, we shouldn't overwrite it. We should check for its presence too before assigning `this._labelId` to it.

Reply:
(This can be done by just moving this line to before `{...inputProps}`)

Reply:
(And also another good thing to add a snapshot test for... sorry for all the comments.)